Suggested edits & comments - forms CMS governance

jenniferlee-dsva opened the pull request:
@beeyayjay - made a few suggested edits & comments, as a branch for tracking changes.

jenniferlee-dsva committed:
Suggested edits & comments - forms CMS governance

jenniferlee-dsva committed:
Update va-forms-cms-governance.md

beeyayjay commented on products/find-va-forms-redesign/form-detail-landing-pages/WIP-technical-specs-and-docs/va-forms-cms-governance.md:
```diff
@@ -1,27 +1,31 @@
-- Because basic form data will be maintained in the Forms DB, non-admin users will not have permission to create or delete Form content in the CMS.
-- Users with appropriate roles and administration membership will have permission to edit Forms pages, but will not be able to edit fields that come from Forms DB.
+- Because basic form data will be maintained in the VA Forms DB, CMS users will not have the ability to create or delete form detail landing pages in the CMS. [@beth - not sure if I understood what this means, so I took a stab based on what I think I understand and what I would expect./jen] 
+- Users with appropriate roles and administration membership will have permission to edit form detail pages, but will not be able to edit fields that come from Forms DB. [Not sure about this. In stage 1, only PW content team with CMS access will have be able to edit or publish form detail pages; so "administration membership" doesn't apply, if I think this is what it means. For example: if a stak ...
```
I think I used the wrong word here ('administration'). What would you call Public Websites in this context? Section, or team, maybe?

jenniferlee-dsva replied:
PW content team. Like this...? 
"PW content team will have permission to edit form detail pages."  <--I'm assuming that any PW content person with editing permission can make edits to the editable fields; and that only the PW content people who have 'publish' permission can both make edits and publish updates. (Some PW content people will have edit access but not publish access.)

jenniferlee-dsva replied:
PW content editor can edit form detail landing pages as 'draft,' but cannot publish. 

PW content publisher can edit, publish, and delete form detail landing pages.

beeyayjay commented on products/find-va-forms-redesign/form-detail-landing-pages/WIP-technical-specs-and-docs/va-forms-cms-governance.md:
```diff
@@ -1,27 +1,31 @@
 
-- Because basic form data will be maintained in the Forms DB, non-admin users will not have permission to create or delete Form content in the CMS.
-- Users with appropriate roles and administration membership will have permission to edit Forms pages, but will not be able to edit fields that come from Forms DB.
+- Because basic form data will be maintained in the VA Forms DB, CMS users will not have the ability to create or delete form detail landing pages in the CMS. [@beth - not sure if I understood what this means, so I took a stab based on what I think I understand and what I would expect./jen] 
```
'Admin' is a special role, reserved mostly for developers. Admins can basically do anything, so these permissions don't apply to them. All other CMS users (non-admins) won't be able to create or delete Forms pages.

jenniferlee-dsva replied:
We might need to talk this one out. Looks really critical. I want to make sure this is set right. Because sometimes even I get an error message saying I'm not an admin with access to a page, even though I am supposed to have global UX admin rights. 

1/ I've seen on the CMS permissions spreadsheet from Howie that there are "Administrators" and also "User Admins" -- are these different? 

2/ Why does anyone need to create a form detail page - that should be auto generated when a new form is added to the VA forms db, no? (Now in the future we may/will want to add a suppress feature for non-VA forms, but for now.) 

3/ For deleting a form detail page - this is tricky and has some downstream things that need to be considered - like redirects, potential for 404s, and current behavior of form detail landing pages (they are not currently automatically removed when a form is deleted from the DB).

jenniferlee-dsva replied:
per convo - Content Publisher should be able to update; publish; and delete form detail landing pages. (This is a change from Administrator.) 

per convo - When forms are migrated into the CMS, it will automatically generate a form detail landing page in **draft** state. (This is a change in understanding from 'publish' state.)

beeyayjay commented on products/find-va-forms-redesign/form-detail-landing-pages/WIP-technical-specs-and-docs/va-forms-cms-governance.md:
```diff
@@ -1,27 +1,31 @@
 
 ## Governance in Stage 2
-We anticipate that various parts of VA will manage the basic form content in Phase 2, while the VSA Public Websites team will retain ownership of additional fields. However Phase 2 governance plans have not yet been finalized. Rather than risk over-engineering, the initial content model will only reflect Phase 1 governance requirements.
+We anticipate that various parts of VA will manage the basic form content in Phase 2, while the Public Websites team will retain ownership of additional fields. (Possible example: VHA form managers could upload new forms or updated versions of forms directly into the CMS; or directly remove retired/expired forms from the CMS.) However Phase 2 governance plans have not yet been finalized. Rather t ...
```
Looks great!

beeyayjay commented on products/find-va-forms-redesign/form-detail-landing-pages/WIP-technical-specs-and-docs/va-forms-cms-governance.md:
```diff
@@ -1,27 +1,31 @@
 Form pages will be governed by the same workflow as other content ('Draft', 'In review' 'Published').
 
-### Permissions
+### CMS permissions
```
Good catch!

jenniferlee-dsva committed:
Updated with new clarification

Updates re migration and draft state for form detail landing pages; difference btn PW content editor and content publisher permissions for form detail landing pages.

jenniferlee-dsva committed:
Update va-forms-cms-governance.md

jenniferlee-dsva committed:
Update va-forms-cms-governance.md
